tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Gather needed facts
+  setup:
+    gather_subset:
+      - '!all'
+      - '!min'
+      - distribution_major_version
+  when: ansible_facts.distribution_major_version is not defined
+
+- name: Enable fapolicyd
+  include_tasks: enable.yml
```

Comment:
You can put all tasks from `enable.yml` here for better readability.

Reply:
I will do that after review so it's clear.